### PR TITLE
Helm release - 1.4

### PR DESCRIPTION
**Pull request by pammecrandall**

### CODE REVIEW

File: helm-chart/configs/grafana-top-dashboard.json [hunk 1]
```diff
@@ -310,7 +310,7 @@
           "alertThreshold": true
         },
         "percentage": false,
-        "pluginVersion": "8.1.7",
```
Comment (kate-osborn) [inline helm-chart/configs/grafana-top-dashboard.json:313]: [this grafana dashboard ](https://github.com/nginxinc/nginx-service-mesh/blob/main/examples/grafana/nginx-service-mesh-summary.json) needs to be updated as well

**Comment by jbyers19** [comment]
`examples/grafana/nginx-mesh-top.json` and `examples/grafana/nginx-service-mesh-summary.json` need to be updated too.

File: helm-chart/configs/grafana-top-dashboard.json (hunk 1 above)
Comment (pammecrandall) [reply helm-chart/configs/grafana-top-dashboard.json:313]: Okay.  Thanks!

Comment (pammecrandall) [reply helm-chart/configs/grafana-top-dashboard.json:313]: I just updated all of the pluginVersion fields in those files.

Comment (jbyers19) [reply helm-chart/configs/grafana-top-dashboard.json:313]: I'm sorry, missed one: `examples/grafana/nginx-mesh-top.json`

Comment (pammecrandall) [reply helm-chart/configs/grafana-top-dashboard.json:313]: Oh.  Thanks!

Comment (pammecrandall) [reply helm-chart/configs/grafana-top-dashboard.json:313]: Can you tell me which line I missed?  I don't see it.

Comment (jbyers19) [reply helm-chart/configs/grafana-top-dashboard.json:313]: Oh, strange. Guess something failed to update on my end. It showed the old version before, but now it's showing the correct one. Sorry about that!

**Review by jbyers19 — APPROVED** [review]

File: helm-chart/configs/grafana-top-dashboard.json (hunk 1 above)
Comment (pammecrandall) [reply helm-chart/configs/grafana-top-dashboard.json:313]: Oh good.  Thanks!

**Pull request merged**